tests/network/services.go

```diff
@@ -240,20 +240,20 @@ var _ = SIGDescribe("Services", func() {
 		)
 
 		createReadyVMIWithMasqueradeBindingAndExposedService := func(hostname string, subdomain string) *v1.VirtualMachineInstance {
-			vmi := libvmi.NewAlpineWithTestTooling(
+			vmi := libvmi.NewFedora(
 				libvmi.WithMasqueradeNetworking()...,
 			)
 			return readyVMI(
 				exposeExistingVMISpec(vmi, subdomain, hostname, selectorLabelKey, selectorLabelValue),
-				console.LoginToAlpine)
+				console.LoginToFedora)
 		}
 
 		BeforeEach(func() {
 			subdomain := "vmi"
 			hostname := "inbound"
 
 			inboundVMI = createReadyVMIWithMasqueradeBindingAndExposedService(hostname, subdomain)
-			tests.StartTCPServer(inboundVMI, servicePort, console.LoginToAlpine)
+			tests.StartTCPServer(inboundVMI, servicePort, console.LoginToFedora)
```

Comment:
@AlonaKaplan  @EdDev Please have a look here as well

Reply:
I think we can pay the price of using a heavier OS here, but it will be good to understand why Alpine is not supporting ARM64.

I am seeing Alpine for ARM exists [here](https://github.com/docker-library/official-images#architectures-other-than-amd64).

Reply:
Yes, Alpine itself does support ARM, but the specific "alpine-with-test-tooling" image used in the e2e tests doesn't currently support Arm64. The build tools for "alpine-with-test-tooling" only accommodate the creation of x86_64 architecture Alpine VM images.
https://github.com/kubevirt/kubevirtci/blob/94f721404b447691d7dac1cb6b00d44b4891dd75/cluster-provision/images/vm-image-builder/alpine-cloud-init/create-image.sh#L4-L6

Reply:
Thank you for the clarification.
Please note that https://github.com/alpinelinux/alpine-make-vm-image/issues/10 has been resolved recently, please consider as a follow-up to support the Alpine custom build images.

Reply:
> Please note that https://github.com/alpinelinux/alpine-make-vm-image/issues/10 has been resolved recently, please consider as a follow-up to support the Alpine custom build images.

I did not notice the issue has been resolved. Thanks for informing. I will try to build Arm64 arch `alpine-with-test-tooling` image.

> If we need to go with the labels (instead of skipping on kind), I could work on introducing the labels if it will help you.

I think this pertains more to the subsequent comments. I will answer you there.